backend/routes/performance.py

```
@@ -0,0 +1,398 @@
+
+Provides detailed performance metrics including:
+- Response time metrics and percentiles
+- Query throughput analysis  
+- Timeline data for charts
+- Error rate tracking
+"""
+
+import logging
+import sqlite3
+from datetime import datetime, timedelta
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)